.gitignore

```diff
@@ -1,194 +1,6 @@
-### Windows template
```

Comment:
Is there a reason for removing all of this?

Reply:
Not really needed with what files are in there at the moment.

Reply:
Well is a generic template because the OSs generate the automatically and they shouldn't be committed